README.md

```diff
@@ -19,3 +19,18 @@ most important part. Add your answer to this markdown file.
 
 Implement a function to convert an adjacency list to an adjacency matrix and
 analyze it as above.
+
+## Plagarism Statement
+
+All exercises must contain the following statement:
+“I certify that I have listed all sources used to complete this exercise, including the use
+of any Large Language Models. All of the work is my own, except where stated
+otherwise. I am aware that plagiarism carries severe penalties and that if plagiarism is
+suspected, charges may be filed against me without prior notice.”
+
+## Note
+I did not complete this excersise last semester, so I am attempting this completely from scratch. I did discuss this with Lily Brongo and then referenced Ishita Patel's repository from last semester to confirm my logic for the code and response before submission.
```

Comment:
Did you start by writing the code, or by referencing the other repository?

Reply:
How much of the code is your own vs how much came from another repository?

Reply:
The inner if statement is the only part that was drastically changed from my first iteration, iterating for the rows and columns was easy enough, the the checking for edges confused me somewhat, so I looked at her code for her check in the edge check in the if statement.

Reply:
I can mark it in the code itself if it makes it easier to see